api_app/analyzers_manager/observable_analyzers/whoisripe.py

```diff
@@ -0,0 +1,34 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+
+from api_app.analyzers_manager import classes
+
+from tests.mock_utils import if_mock_connections, patch, MockResponse
+
+class Whoisripeapi(classes.ObservableAnalyzer):
```

Comment:
rename to `WhoIsRipeAPI`. change the same in python_module and docs too.

Reply:
Understood.